smart-contracts/contracts/Unlock.sol

```diff
@@ -26,13 +26,13 @@ pragma solidity 0.4.24;
  *  b. Keeping track of GNP
  */
 
-import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-eth/contracts/ownership/Ownable.sol";
 import "zos-lib/contracts/Initializable.sol";
 import "./PublicLock.sol";
 import "./interfaces/IUnlock.sol";
 
 
-contract Unlock is IUnlock, Ownable, Initializable {
+contract Unlock is IUnlock, Initializable, Ownable {
```

Comment:
I had to reorder to fix an error.

Reply:
" TypeError: Linearization of inheritance graph impossible "

Reply:
I could do that. The compiler won't let anyone change this on us by accident, it just won't compile.
https://solidity.readthedocs.io/en/latest/contracts.html?highlight=linearization#multiple-inheritance-and-linearization

Reply:
The compiler won't let it be written any other way
"the order in which the base classes are given in the is directive is important: You have to list the direct base contracts in the order from “most base-like” to “most derived”. 
https://solidity.readthedocs.io/en/latest/contracts.html?highlight=linearization#multiple-inheritance-and-linearization

Reply:
Done.